developer_toolbox/lint_features.json.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
-features=config/retrodeck/reference_lists/features.json
+config="/app/retrodeck/config"  
+features=$config/retrodeck/reference_lists/features.json
```

Comment:
I believe that this was correct before as it was poiting to the repo's features.json.

Reply:
On the cooker build I tested it was not working and the json file was not getting copied?

I will double check again against this cooker build I was testing

https://github.com/RetroDECK/Cooker/releases/tag/cooker-0.9.0b-ShowTale

Reply:
> On the cooker build I tested it was not working and the json file was not getting copied?
> 
> I will double check again against this cooker build I was testing
> 
> https://github.com/RetroDECK/Cooker/releases/tag/cooker-0.9.0b-ShowTale

Yes it's still working as it's a developer toolbox tool that is not getting inside the build.